bottlerocket-agents/src/bin/ec2-resource-agent/ec2_provider.rs

```diff
@@ -149,9 +149,17 @@ impl Create for Ec2Creator {
             );
 
         info!("Starting instances");
-        let instances = run_instances
-            .send()
-            .await
+        let mut run_instance_result = run_instances.clone().send().await;
+        if let Err(SdkError::ServiceError { err, raw: _ }) = &run_instance_result {
+            if matches!(&err.code(), Some("InvalidParameterValue")) {
+                info!("An error occured while trying to run instances. Will retry in 60s.");
```

Comment:
maybe have this at warning level and log the error message as well?
```suggestion
                warn!("An error occured while trying to run instances: '{}'. Retrying in 60s.", err);
```